### PR TITLE
SettingUpReactHooks storybook page update

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
# What

# Why

# How Tested

storybook

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/Stateful Client/SettingUpReactHooks.stories.mdx [hunk 1]
```diff
@@ -60,4 +60,4 @@ ReactDOM.render(
 
 Once you have wrapped your entire app with providers, you are now can use the usePropsFor() method inside of your app to connect your components to the state.
 
-To see a full end-to-end example using providers, see [Getting Started with StatefulClient](./?path=/docs/quickstarts-statefulclient--page)
+To see a full end-to-end examples using providers, see [Getting Started with Call Stateful Client](./?path=/docs/quickstarts-statefulcallclient--page) and [Getting Started with Chat Stateful Client](./?path=/docs/quickstarts-statefulchatclient--page).
```
Comment (emlynmac) [inline packages/storybook/stories/Stateful Client/SettingUpReactHooks.stories.mdx:63]: examples should be example,
```suggestion
To see a full end-to-end example using providers, see [Getting Started with Call Stateful Client](./?path=/docs/quickstarts-statefulcallclient--page) and [Getting Started with Chat Stateful Client](./?path=/docs/quickstarts-statefulchatclient--page).
```
Match singularity of 'a'

**Review by emlynmac — APPROVED** [review]
one minor comment only.
Is there a v8 page that needs updating also?

File: packages/storybook/stories/Stateful Client/SettingUpReactHooks.stories.mdx (hunk 1 above)
Comment (vhuseinova-msft) [reply packages/storybook/stories/Stateful Client/SettingUpReactHooks.stories.mdx]: Removed `a`, thanks!

**Comment by vhuseinova-msft** [comment]
> one minor comment only. Is there a v8 page that needs updating also?

No, this page wasn't migrated yet

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**